### DISCUSSION TITLE
Cannot add a new project on hobby plan for a repository owned by Github Organization

### DISCUSSION THREAD

**Discussion by claudio-darkkenergy** [vercel/community#6733] [Account Issue]
### Question

I'm having trouble creating a new project for my Hobby plan using a Github Repository from a Github Organization. This is confusing to me, because I've already created one project in this way, "loomjs-docs". And, I verified that I should be allowed to create up to 3 projects on a Hobby Plan.

However, I also understand that I shouldn't be able to even create a new project in this way, but again, why was I able to before? I'm just trying to set up a new project for my monorepo using the same repository, but pointing to a different directory.

The confusion is how I even came to have a project in this way, which is apparently not allowed. Did something change along the way?

This post is similar to [this post](https://github.com/orgs/vercel/discussions/6645), but a reason wasn't really provided there.

<img width="1406" alt="Screenshot 2024-05-04 at 9 14 37 PM" src="https://github.com/vercel/community/assets/113791849/9cfbdc71-61cf-4342-9490-ca15960cd163">
<img width="598" alt="Screenshot 2024-05-04 at 9 15 15 PM" src="https://github.com/vercel/community/assets/113791849/0f7afa6d-e871-4738-8e4e-cfd6f99d1dfa">

**Comment by leerob** [comment]
Hey! You will need to upgrade to a paid account (Pro). Nothing has changed here, it's been this way.

x-ref: https://github.com/orgs/vercel/discussions/6645#discussioncomment-9240168

**Reply by claudio-darkkenergy** [reply]
Apologies, but perhaps my question was a bit vague. And thank you for your response, as well - that part is clear to me. And this answer is basically what I've already found in [this post](https://github.com/orgs/vercel/discussions/6645).

Referencing back to two of my statements for which I'm truly looking for an answer which is not clear to me. 

> This is confusing to me, because I've already created one project in this way,
> why was I able to before?

I did create a project for a Github repository which is owned by a Github Organization, under my Hobby Plan. I'm just looking for clarification on how this could have happened since it's not permitted. I've been using this project for quite some time.

Or, am I allowed one project set up in this way? This is what doesn't make sense to me, and I'm just trying to make sense of it.

Thanks again!
